components/sensors.json

```diff
@@ -51,6 +51,14 @@
     "displayName": "Thermocouple Temperature",
     "defaultPeriod": 900
   },
+  "ambient-temp-fahrenheit": {
+    "displayName": "Temperature Sensor (degrees Fahrenheit)",
+    "defaultPeriod": 900
+  },
+  "object-temp-fahrenheit": {
+    "displayName": "Thermocouple Temperature (degrees Fahrenheit)",
```

Comment:
Awareness: longish display names, will make forms ugly/busy.

Maybe  `°F` instead of typing it out? maybe add `°C` to the other ones?